drivers/i3c/master/mipi-i3c-hci/pio.c

```diff
@@ -693,7 +693,7 @@ static bool hci_pio_process_resp(struct i3c_hci *hci, struct hci_pio_data *pio)
 				hci_pio_push_to_next_rx(hci, xfer, to_keep);
 			}
 			/* Workaround for A0 dummy data issue */
-			aspeed_dummy_data_work_around(hci, pio);
+			// aspeed_dummy_data_work_around(hci, pio);
```

Comment:
Use revert commit [f94bbde](https://github.com/AspeedTech-BMC) instead of manually disabling the feature, to ensure changes are properly tracked and managed through version control.